components/raftstore/src/store/fsm/apply.rs

```diff
@@ -707,22 +703,22 @@ struct WaitSourceMergeState {
     logs_up_to_date: Arc<AtomicU64>,
 }
 
-struct YieldState<E>
+struct YieldState<EK>
 where
-    E: KvEngine,
+    EK: KvEngine,
 {
     /// All of the entries that need to continue to be applied after
     /// the source peer has applied its logs.
     pending_entries: Vec<Entry>,
     /// All of messages that need to continue to be handled after
     /// the source peer has applied its logs and pending entries
     /// are all handled.
-    pending_msgs: Vec<Msg<E>>,
+    pending_msgs: Vec<Msg<EK>>,
```

Comment:
Is it necessary to replace all 'E' with 'EK'?

Reply:
I prefer `EK` and `ER` so that in the next PR I can replace all `ER` easilier.